examples/lzapp-migration/tasks/common/config.get.ts

```diff
@@ -0,0 +1,257 @@
+                    receiveUlnConfig: receiveOAppUlnConfig,
+                },
+            }
+
+            console.log(
+                printCrossTable(
+                    [
+                        {
+                            localNetworkName,
+                            remoteNetworkName,
+                            sendLibrary: sendCustomLibrary,
+                            receiveLibrary: receiveCustomLibrary,
+                            sendUlnConfig: sendCustomUlnConfig,
+                            sendExecutorConfig: sendCustomExecutorConfig,
+                            receiveUlnConfig: receiveCustomUlnConfig,
+                        },
+                        {
+                            localNetworkName,
+                            remoteNetworkName,
+                            sendLibrary: sendDefaultLibrary,
+                            receiveLibrary: receiveDefaultLibrary,
+                            sendUlnConfig: sendDefaultUlnConfig,
+                            sendExecutorConfig: sendDefaultExecutorConfig,
+                            receiveUlnConfig: receiveDefaultUlnConfig,
+                        },
+                        {
+                            localNetworkName,
+                            remoteNetworkName,
+                            sendLibrary: sendOAppLibrary,
+                            receiveLibrary: receiveOAppLibrary,
+                            sendUlnConfig: sendOAppUlnConfig,
+                            sendExecutorConfig: sendOAppExecutorConfig,
+                            receiveUlnConfig: receiveOAppUlnConfig,
+                        },
+                    ],
+                    ['', 'Custom OApp Config', 'Default OApp Config', 'Active OApp Config']
+                )
+            )
+        })
+
+    const epv1Tasks = graph.connections
+        .filter(({ vector: { from } }) => isEndpointV1Eid(from) && isEvmEndpoint(from))
+        .map(({ vector: { from, to } }) => async () => {
+            const hreForEid = await getHreByEid(from.eid)
+            if (!hreForEid) {
+                logger.error(`Failed to obtain hre for EID ${from.eid}`)
+                return
+            }
+
+            // Fetch configurations based on EID version
+            if (isEndpointV1Eid(from)) {
+                // Active configurations
+                const executorConfig = await getEpv1ExecutorConfig(hreForEid, to.eid, from.address)
+                const sendUlnConfig = await getEpv1SendUlnConfig(hreForEid, to.eid, from.address)
+                const sendLibraryAddress = await getEpv1SendLibraryAddress(hreForEid, from.address)
+                const receiveUlnConfig = await getEpv1ReceiveUlnConfig(hreForEid, to.eid, from.address)
+                const receiveLibraryAddress = await getEpv1ReceiveLibraryAddress(hreForEid, to.eid, from.address)
+
+                // if (!executorConfig || sendUlnConfig || !receiveUlnConfig || !sendLibraryAddress || !receiveLibraryAddress) {
```

Comment:
Why was this commented out?  I think you missed a `!` for `sendUlnConfig`.  Wondering if we should keep this in?